src/apis/collectives/urls.js

```diff
@@ -34,3 +34,11 @@ export function pagesUrl(context, ...parts) {
 		? collectivesUrl('p', context.shareTokenParam, '_pages', ...parts)
 		: collectivesUrl(context.collectiveId, '_pages', ...parts)
 }
+
+/**
+ *
+ * @param {number} collectiveId collectiveId to search in
+ */
+export function getContentSearchUrl(collectiveId) {
```

Comment:
```suggestion
export function contentSearchUrl(collectiveId) {
```

All the other functions in `urls.js` don't have `get` as prefix, so maybe stick to that for now?